src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -175,39 +196,58 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
-                                        childAssets={this.state.childAssets} />
-                                </Canvas>
-                            }
-                        </div>
-                        <div>
-                            <EditorFooter
-                                tags={this.props.project.tags}
-                                lockedTags={this.state.lockedTags}
-                                onTagsChanged={this.onFooterChange}
-                                onTagClicked={this.onTagClicked}
-                                onCtrlTagClicked={this.onCtrlTagClicked}
-                            />
+                            <div className="editor-page-content-body-canvas">
+                                {selectedAsset &&
+                                    <Canvas
+                                        ref={this.canvas}
+                                        selectedAsset={this.state.selectedAsset}
+                                        onAssetMetadataChanged={this.onAssetMetadataChanged}
+                                        onSelectedRegionsChanged={(selectedRegions) => this.setState({selectedRegions})}
+                                        editorMode={this.state.editorMode}
+                                        selectionMode={this.state.selectionMode}
+                                        project={this.props.project}
+                                        lockedTags={this.state.lockedTags}>
+                                        <AssetPreview
+                                            additionalSettings={this.state.additionalSettings}
+                                            autoPlay={true}
+                                            onChildAssetSelected={this.onChildAssetSelected}
+                                            asset={this.state.selectedAsset.asset}
+                                            childAssets={this.state.childAssets} />
+                                    </Canvas>
+                                }
+                            </div>
+                            <div className="editor-page-right-sidebar">
+                                <TagInput
+                                    tags={this.props.project.tags}
+                                    lockedTags={this.state.lockedTags}
+                                    selectedRegions={this.state.selectedRegions}
+                                    onChange={this.onTagsChanged}
+                                    onLockedTagsChange={this.onLockedTagsChanged}
+                                    onTagClick={this.onTagClicked}
+                                    onCtrlTagClick={this.onCtrlTagClicked}
+                                />
+                            </div>
                         </div>
                     </div>
                 </SplitPane>
             </div>
         );
     }
 
+    private getTarget = () => {
```

Comment:
Still in use?  I don't believe you are using this anymore.

Reply:
Also unused